src/docs/content/observability/observability-scenario/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Observability Scenario"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+showChildren: true
+---
+
+## Introduction
+
+In this section we will try to understand Observability on Oracle Cloud Infrastructure (OCI) by simulating a failure scenario on MuShop application.
+
+![Failure-scenario](../images/observability-scenario.png)
+
+## Scenario Details
+- User logs in to MuShop and Orders items with value more than $105.
+- User notices the Order processing failure.
+- User gets notified on the Payment failures. 
+- User analyzes the failure using Oracle Cloud Infrastructure(OCI) Observability Services.
+
+## Following topics will be covered
```

Comment:
I suggest just combining into a single file for the whole example

Reply:
combined created 2 files : prereqs in setup.md and example in usecase.md.